frontend/src/pages/accounts/[id].tsx

```diff
@@ -15,9 +14,10 @@ import Content from "../../components/utils/Content";
 import TransactionIcon from "../../../public/static/images/icon-t-transactions.svg";
 
 import { Translate } from "react-localize-redux";
-import { NextPageContext } from "next";
+import { NextPage } from "next";
+import { useNearNetwork } from "src/hooks/use-near-network";
```

Comment:
1. These do not follow the contribution guide. Let's move the imports into the appropriate groups.
2. Should we change all relative imports that reach `src` folder to `src/` imports? (we should mention that in the contributing guide)

Reply:
I believe there is only one way that has minimal frustration to enforce any rules on import order / styles - linter (with fix).
Should I make it as in contribution guide in this PR or should we wait until I'll incorporate linter with import-order rule?